docs/labwc-theme.5.scd

```diff
@@ -196,8 +196,9 @@ elements are not listed here, but are supported.
 	boxes. Default is 20.
 
 *border.color*
-	Set all border colors. This is obsolete, but supported for backward
-	compatibility as some themes still contain it.
+	Set both *window.active.border.color* and *window.active.border.color*.
```

Comment:
I assume one of them should read `window.inactive.border.color`.

Reply:
Fixed.